core/src/main/java/com/opensymphony/xwork2/config/impl/DefaultConfiguration.java

```diff
@@ -284,7 +284,7 @@ protected Container createBootstrapContainer(List<ContainerProvider> providers)
         builder.factory(ValueSubstitutor.class, EnvsValueSubstitutor.class, Scope.SINGLETON);
 
         builder.constant(StrutsConstants.STRUTS_DEVMODE, "false");
-        builder.constant(StrutsConstants.STRUTS_LOG_MISSING_PROPERTIES, "false");
+        builder.constant(StrutsConstants.STRUTS_OGNL_LOG_MISSING_PROPERTIES, "false");
```

Comment:
Does this mean, the OGNL log constant is preferred over the non OGNL constant?

Reply:
Oh, I see it is a rename under https://github.com/apache/struts/pull/464/files#diff-2de2f780034a74f8de7c95a0a38f6bea1c97cff74d7060064f28dc6069a24a44

Reply:
Yes, just a rename to match the same pattern.